.travis.yml

```diff
@@ -1,16 +1,26 @@
 language: julia
+
 os:
     - linux
     - osx
+
 julia:
-    - 0.5
+    - 0.6
     - nightly
+
+matrix:
+    allow_failures:
```

Comment:
Showing a big fat red "BUILD FAILING"-badge as the first thing a user sees because Base has changed is lame :P. It is, of course, good to have the package work on master but failing the build is excessive in my opinion.